### PR TITLE
Support Java performance worker in run_performance_tests.py

### PR DESCRIPTION
**Pull request by jtattermusch**

### CODE REVIEW

File: tools/run_tests/performance/scenario_config.py [hunk 1]
```diff
@@ -31,6 +31,7 @@
 
 SINGLE_MACHINE_CORES=8
 WARMUP_SECONDS=5
+JAVA_WARMUP_SECONDS=15  # Java needs more warmup time for JIT to kick in.
```
Comment (vjpai) [inline tools/run_tests/performance/scenario_config.py]: I agree that this is the best we can do for now, but Java really needs a 2-phase warmup. First at load for a period, then quiescent for a period (to give the JIT a chance to work). This needs driver support. I'll file an issue for this.

**Comment by vjpai** [comment]
LGTM.

**Comment by jtattermusch** [comment]
Rebased to resolve conflicts, waiting for tests.

**Comment by vjpai** [comment]
LGTM.

**Pull request merged**